app/view_models/admin/draft_list_item.rb

```diff
@@ -15,7 +15,7 @@ def id
 
   def c2_proposal_status
     if auction.purchase_card == 'default'
-      c2_proposal_status_presenter.status
+      I18n.t("drafts.c2_status.#{auction.c2_status}.status")
```

Comment:
I was wondering why we didn't do this earlier. I had assumed we were being explicit for a reason instead of doing conventions vs. configuration